src/templates/twoStageSurvey.html

```diff
@@ -63,39 +69,12 @@ <h4 class="bolded-blue" data-i18n="moral-training-task1"></h4>
         </div>
 
         <div id="group" class="mt-5">
-            <h4 class="bolded-blue" data-i18n="moral-training-task2-{{topic}}"></h4>
+            <h4 class="bolded-blue" data-i18n="moral-training-task2-controversy"></h4>
+            <h5 class="bolded-blue" data-i18n="moral-training-task2-controversy-info"></h5>
 
             <div class="fs-4" style="line-height: 1.1;">
-                
```

Comment:
that should be the content of Survey 1.

Did you make sure there is nothing changed to it?